spec/models/task_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :model do
+  describe 'DB table' do
+    it { is_expected.to have_db_column :id }
+    it { is_expected.to have_db_column :title }
+    it { is_expected.to have_db_column :description }
+    it { is_expected.to have_db_column :location }
+    it { is_expected.to have_db_column :budget }
+  end
+
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :description }
+    it { is_expected.to validate_presence_of :location }
+    it { is_expected.to validate_presence_of :budget } 
+  end
+
+  describe 'Factory' do
+    it 'should have valid Factory' do
+      expect(FactoryBot.create(:task)).to be_valid 
```

Comment:
Please include the FactoryBot matchers to your `rails_helper` to shorten this. 

```
config.include FactoryBot::Syntax::Methods
```
And now you can do 
```
expect(create(:task)).to be_valid 
```